### ISSUE TITLE
vscode extension infinite loading bug

### ISSUE THREAD

**Issue by raymondanythings** [rangav/thunder-client-support#1241]
**Describe the bug**

I've been using the Thunder Client extension on VSCode. As of July 29, 2023, the extension has stopped loading.

Here are the things I have tried:

1. Reinstalling the extension
2. Reinstalling VSCode
3. Rebooting my PC

The issue still persists. Do you have any clues to help resolve this situation?

**To Reproduce**

**Expected behavior**

https://github.com/rangav/thunder-client-support/assets/73725736/6b5a2848-cf82-4399-a5ac-7ac6f798643f

**Platform:** 
 - OS: MacOS : 13.4.1
 - vscode version: 1.79.0
 - extension version:v2.9.3

**Your Team Size Using TC:**

1

**Comment by rangav** [comment]
Hi @raymondanythings 

try the following:

1. Install previous version of the extension and try again 
2. I assume the data is stored in default location:
   - please back up all the json files and folders 
   - Delete all files in default location - https://github.com/rangav/thunder-client-support#storage

**Comment by raymondanythings** [comment]
@rangav 
As you advised, I reinstalled the extension version 2.9.2 and confirmed it is now operating normally! 
During the troubleshooting, I found that there was no folder corresponding to the thunder-client in the Default Storage Location. I never deleted it on my own, so I'm assuming it was deleted during the use of the extension. 
Thank you for your help in solving this problem. I hope this case will be helpful.
